tutorials/Numerics/TimeStepping/tutorial_risingbubble_config.jl

```diff
@@ -0,0 +1,316 @@
+using CLIMAParameters.Planet: R_d, cp_d, cv_d, MSLP, grav
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet();
+
+# ## [Initial Conditions](@id init-rtb)
+# This example demonstrates the use of functions defined
+# in the [`Thermodynamics`](@ref ClimateMachine.Thermodynamics) package to
+# generate the appropriate initial state for our problem.
+
+#md # !!! note
+#md #     The following variables are assigned in the initial condition
+#md #     - `state.ρ` = Scalar quantity for initial density profile
+#md #     - `state.ρu`= 3-component vector for initial momentum profile
+#md #     - `state.ρe`= Scalar quantity for initial total-energy profile
+#md #       humidity
+#md #     - `state.tracers.ρχ` = Vector of four tracers (here, for demonstration
+#md #       only; we can interpret these as dye injections for visualization
+#md #       purposes)
+function init_risingbubble!(problem, bl, state, aux, localgeo, t)
+    (x, y, z) = localgeo.coord
+
+    ## Problem float-type
+    FT = eltype(state)
+
+    ## Unpack constant parameters
+    R_gas::FT = R_d(bl.param_set)
+    c_p::FT = cp_d(bl.param_set)
+    c_v::FT = cv_d(bl.param_set)
+    p0::FT = MSLP(bl.param_set)
+    _grav::FT = grav(bl.param_set)
+    γ::FT = c_p / c_v
+
+    ## Define bubble center and background potential temperature
+    xc::FT = 5000
+    yc::FT = 1000
+    zc::FT = 2000
+    r = sqrt((x - xc)^2 + (z - zc)^2)
+    rc::FT = 2000
+    θamplitude::FT = 2
+
+    ## This is configured in the reference hydrostatic state
+    θ_ref::FT = bl.ref_state.virtual_temperature_profile.T_surface
+
+    ## Add the thermal perturbation:
+    Δθ::FT = 0
+    if r <= rc
+        Δθ = θamplitude * (1.0 - r / rc)
+    end
+
+    ## Compute perturbed thermodynamic state:
+    θ = θ_ref + Δθ                                      ## potential temperature
+    π_exner = FT(1) - _grav / (c_p * θ) * z             ## exner pressure
+    ρ = p0 / (R_gas * θ) * (π_exner)^(c_v / R_gas)      ## density
+    T = θ * π_exner
+    e_int = internal_energy(bl.param_set, T)
+    ts = PhaseDry(bl.param_set, e_int, ρ)
+    ρu = SVector(FT(0), FT(0), FT(0))                   ## momentum
+    ## State (prognostic) variable assignment
+    e_kin = FT(0)                                       ## kinetic energy
+    e_pot = gravitational_potential(bl, aux)            ## potential energy
+    ρe_tot = ρ * total_energy(e_kin, e_pot, ts)         ## total energy
+
+    ρχ = FT(0)                                          ## tracer
+
+    ## We inject tracers at the initial condition at some specified z coordinates
+    if 500 < z <= 550
+        ρχ += FT(0.05)
+    end
+
+    ## We want 4 tracers
+    ntracers = 4
+
+    ## Define 4 tracers, (arbitrary scaling for this demo problem)
+    ρχ = SVector{ntracers, FT}(ρχ, ρχ / 2, ρχ / 3, ρχ / 4)
+
+    ## Assign State Variables
+    state.ρ = ρ
+    state.ρu = ρu
+    state.ρe = ρe_tot
```

Comment:
This has changed on master. @charleskawczynski what should be done here?

Reply:
`state.energy.ρe = ρe_tot`

Reply:
Thanks!